pkg/debian/upstream/metadata

```diff
@@ -0,0 +1,7 @@
+Bug-Database: https://github.com/Tarsnap/tarsnap/issues
+Bug-Submit: https://github.com/Tarsnap/tarsnap/issues/new
+Bug-Submit: https://www.tarsnap.com/support.html
```

Comment:
Is it ok to have `Bug-Submit` twice?  I can't find a definitive answer as to whether fields can be repeated.

I'd be tempted to remove the second one, or perhaps even rename it to `Support-Page:`.  That's not listed in DEP12, but it does say that one can define new fields.

Reply:
or possibly `Other-Reference:`?  I'm not certain whether that would fit into standard practice.